src/managment/Reducer.tsx

```diff
@@ -0,0 +1,88 @@
+import { State, Filter } from '../types/Types';
+
+export type Action =
+  | { type: 'addTodo'; title: string }
+  | { type: 'removeTodo'; id: number }
+  | { type: 'editTitle'; id: number; newTitle: string }
+  | { type: 'markDone'; id: number }
+  | { type: 'toggleDone'; payload: boolean }
+  | { type: 'removeComplTodos' }
```

Comment:
```suggestion
  | { type: 'removeCompletedTodos' }
```
don't use short forms